src/components/form/DatePicker/DatePicker.js

```diff
@@ -0,0 +1,30 @@
+import { StyledDatePicker } from "./DatePicker.styled";
+
+export const DatePicker = ({
+  field,
+  value,
+  handleChange,
+  errorMessage,
+  isTouched,
+  handleBlur,
+}) => {
+  const currentDate = new Date().toJSON().slice(0, 10);
```

Comment:
you can also create some method in new file for getting this date new Date().toJSON().slice(0, 10);